zerver/views/webhooks/transifex.py

```diff
@@ -0,0 +1,23 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+
+
+@api_key_only_webhook_view('Transifex')
+@has_request_variables
+def api_transifex_webhook(request, user_profile, client,
+                          project=REQ(), resource=REQ(),
+                          language=REQ(), translated=REQ(default=None),
+                          reviewed=REQ(default=None),
+                          stream=REQ(default='transifex')):
+    subject = "{} - language: {}".format(project, language)
```

Comment:
Maybe we should make this a bit more brief by leaving off the word "language", e.g.:

`subject = "{} in {}".format(project, language)` which would display as e.g. "zulip in ru"?

Otherwise, this looks great!